ibis/backends/clickhouse/compiler/relations.py

```diff
@@ -17,79 +16,94 @@ def translate_rel(op: ops.TableNode, **_):
     raise com.OperationNotDefinedError(f"No translation rule for {type(op)}")
 
 
-@translate_rel.register(ops.DummyTable)
-def _dummy(op: ops.DummyTable, **kw):
-    return sg.select(
-        *map(partial(translate_val, **kw), op.values), dialect="clickhouse"
-    )
+@translate_rel.register
+def _dummy(op: ops.DummyTable, *, values, **_):
```

Comment:
Can you remove the keyword-only marker `*` from the functions? I usually find it more distracting than useful.